src/config-schema.coffee

```diff
@@ -67,6 +67,10 @@ module.exports =
             type: 'boolean'
             default: false
             title: "Use backspace / delete to move messages to trash"
+          timeFormat:            # military time(24 hour clock) section added here -->
```

Comment:
Can we rename this config from `timeFormat` to something that sounds a little more like a boolean value. When I see a variable like `timeFormat` I expect it to hold a formatting string or some piece of data instead of just a boolean value. Maybe something like `is24HrFormat`